### PR TITLE
chore: Enables gofmt linting

### PR DESCRIPTION
**Pull request by gruyaume**
# Description

This PR enables the `go fmt` lint check and makes the necessary change to satisfy it.

### CODE REVIEW

**Comment by gab-arrobo** [comment]
ok to test

**Review by gab-arrobo — APPROVED** [review]
+1 (tested changes locally)

File: consumer/nf_management.go [hunk 1]
```diff
@@ -39,7 +39,8 @@ func BuildNFInstance(ausfContext *ausf_context.AUSFContext) (profile models.NfPr
 }
 
 // func SendRegisterNFInstance(nrfUri, nfInstanceId string, profile models.NfProfile) (resouceNrfUri string,
-//    retrieveNfInstanceID string, err error) {
+//
+//	retrieveNfInstanceID string, err error) {
```
Comment (gab-arrobo) [inline consumer/nf_management.go:43]: @gruyaume, actually, should we remove all these lines?

File: producer/ue_authentication.go [hunk 2]
```diff
@@ -86,7 +86,8 @@ func HandleUeAuthPostRequest(request *http_wrapper.Request) *http_wrapper.Respon
 }
 
 // func UeAuthPostRequestProcedure(updateAuthenticationInfo models.AuthenticationInfo) (
-//    response *models.UeAuthenticationCtx, locationURI string, problemDetails *models.ProblemDetails) {
+//
+//	response *models.UeAuthenticationCtx, locationURI string, problemDetails *models.ProblemDetails) {
```
Comment (gab-arrobo) [inline producer/ue_authentication.go:90]: @gruyaume, actually, should we remove all these lines?

File: consumer/nf_management.go (hunk 1 above)
Comment (gruyaume) [reply consumer/nf_management.go:43]: Well probably but the goal here was simply to satisfy `gofmt`. All over the code there are comments that are likely not wanted but I didn't want to go through this activity here.

File: producer/ue_authentication.go (hunk 2 above)
Comment (gruyaume) [reply producer/ue_authentication.go:90]: Please read my reply above.

**Pull request merged**